src/theme/icons.scss

```diff
@@ -77,6 +77,10 @@
   color: #464d5f;
 }
 
+.tab-badge {
```

Comment:
It's already fixed here (not sure if it applies): https://github.com/bitpay/copay/pull/10738/files#diff-3c303face59d40ba5031ed6c51449334R793